.github/workflows/publish-stable.yml

```diff
@@ -12,6 +12,28 @@ on:
       - completed
 
 jobs:
+  generate-metadata-for-extension:
+    # Generate metadata for extension before automerging the current branch to master
+    # only run if prechecks pass and the source branch is named 'prerelease/*'
+    if: ${{ startsWith(github.head_ref, 'prerelease/1') }} && ${{ github.event.label.name == 'automerge' }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: generate meta for extension
+      uses: actions/checkout@v2
```

Comment:
this will check out the default branch
you might mean to use a different branch

Reply:
I wanted GitHub Action to use current branch (prerelease one)... will investigate this part..

Reply:
fixed this.. using 
```
with:
 ref: ${{ github.event.pull_request.head.ref }}
```
tested it.. it does commits and pushes my changes to the current branch